Created an Introduction page for the User Guide and the General Settings Section (#2024)
strapi/documentation: docusaurus/docs/user-docs/settings/introduction.md

---
title: Introduction to General Settings
description: Reviewing the audit logs in Strapi.
sidebar_position: 2
---

> The ![Settings icon](/img/assets/icons/settings.svg) _Settings_ section in the main navigation of the admin panel  includes all the required set-up information that determines how an administrator interacts with and manages their Strapi application. 

Reviewer: ```suggestion
The ![Settings icon](/img/assets/icons/settings.svg) _Settings_ section in the main navigation of the admin panel includes all the required set-up information that determines how an administrator interacts with and manages their Strapi application. 
```

Duplicate space :D

---
title: Introduction to General Settings
description: Reviewing the audit logs in Strapi.
sidebar_position: 2
---

The ![Settings icon](/img/assets/icons/settings.svg) _Settings_ section in the main navigation of the admin panel comprises of all the required set-up information that determines how an administrator interact with and manage their Strapi application. It is divided into 4 sub sections:

> - _Global Settings_

Reviewer: I'm not sure we need this list here 🤔 It's not wrong, it just does not feel like the way other pages are organized IMHO. I would suggest you use the following structure:
1. Keep the first sentence ("The settings section … manage their Strapi application"
2. Insert the screenshot.
3. Simply introduce the table with a sentence that mixes the current second sentence of the introduction with the current sentence above the table. Something like: "The following table list the 4 Settings sub-sections, available settings in each sub-section, and corresponding links on this user guide to access detailed information:"

Again, your suggestion is not not wrong (apart from the fact that we prefer to use "the following" and never use "above" or "below" in the docs), it just feels, IMHO, a bit less consistent with other introductory pages in the User Guide, hence my suggestion.
What do you think? 😊

---
title: Introduction to General Settings
description: Reviewing the audit logs in Strapi.
sidebar_position: 2
---

> The _Settings_ feature allows an administrator to adjust all ![Settings icon](/img/assets/icons/settings.svg) _Settings_ from the main navigation of the admin panel, which is divided into 4 sub sections:

Reviewer 2: ```suggestion
The ![Settings icon](/img/assets/icons/settings.svg) _Settings_ section in the main navigation of the admin panel allows an administrator to adjust various settings for the project. It is divided into 4 sub sections:
```

Suggestion of rephrasing because it sounds a bit weird to me to call the "Settings" section a feature. I also think we could go even more quickly to the point with this sentence 😊

- _Global Settings_
- _Administration panel_
- _Email plugin_
- _Users & Permissions plugin_

<ThemedImage
alt="Custom logo settings"
sources={{
    light: '/img/assets/settings/settings_custom-logo.png',
    dark: '/img/assets/settings/settings_custom-logo_DARK.png',
  }}
/>

## Global Settings

> The _Global Settings_ in Strapi allows administrators to configure logo in the _Overview & Custom Logo_ interface, manage _API Token_, manage _Media Library_, configure _Internationalization_ locales and configure _Single Sign-On_ and _Webhooks_.

Reviewer 2: Perhaps we should make this introduction even more helpful by adding links to those pages, what do you think?

## Administration Panel

The _Administration Panel_ Settings in Strapi allows administrators to configure _Roles_ and _Users_.

### Roles

The _Role_ interface allows administrators to edit the details of an administrator role and configure its permissions in detail. It can be accessed from _Administration panel_ > _Roles_ (see [Configuring Administrator Roles](/user-docs/users-roles-permissions/configuring-administrator-roles)).

### Users

> The _Users_ interface allows administrators to edit the details of an end-users role and configure its permissions in detail. It can be accessed from _Administration panel_ > _Users_.

Reviewer 2: I have the feeling that those paragraphs are very repetitive, and I'm not sure they're very necessary since neither are documented in the settings section anyways 🤔 
Perhaps we could just explain briefly what the "Administration Panel" section of settings contains and redirect users to the section of the user guide where everything is already documented?

## Email plugin

> Administrators must configure an email provider and its corresponding settings in the config/plugins.js or config/plugins.ts file to use the _Email plugin_ Settings. By default, Strapi uses **Sendmail** as the email provider, which is suitable for local development but requires further configuration for production environments. See the [Developer Documentation](/dev-docs/plugins/email) for more details.

Reviewer 2: Don't forget to apply the correct formatting when writing files names.

> ## Users & Permission plugin

Reviewer 2: I'm also not super convinced with this section yet. I feel like we're basically re-explaining/re-using what's already documented somewhere else instead of adding new information.

I think we should rethink this introduction together (cc @pwizla) because the idea should probably to explain the logic + more context behind the features in the settings section, and allow users to understand even more e.g. why this particular option is there, what is the main feature behind these options etc.

Reviewer: I agree with Mégane. @MbonuJennifer, I think this page needs a big rewrite. 

The goal is to have a simple/short introduction for settings. Users need to understand:

- Where is the Settings section in the admin panel and what it does include (we don't need too many details as all dedicated sections in the documentation already describe them)
- What they will find in this section? Maybe a simple table with short description for the sections and links to the documentation pages would work?

This introduction should be there just to give a bit more context about settings and guide users through the different settings available and their related documentation.

Author: No worries, thank you I'll rewrite this page with this feedback in mind.
